pyomo/contrib/mindtpy/util.py

```diff
@@ -89,20 +90,44 @@ def calc_jacobians(solve_data, config):
             for var, jac_wrt_var in zip(vars_in_constr, jac_list))
 
 
+# def add_feas_slacks(m, config):
```

Comment:
We can remove these comments since the code below generalizes what is commented.